ci-operator/config/openshift-priv/aws-ebs-csi-driver/openshift-priv-aws-ebs-csi-driver-release-4.5.yaml

```diff
@@ -17,13 +17,6 @@ canonical_go_repository: github.com/kubernetes-sigs/aws-ebs-csi-driver
 images:
 - dockerfile_path: Dockerfile.openshift
```

Comment:
Hm. This is a case where the CI config *should* be using their "RHEL" Dockerfile, and there should be no difference -- from a policy perspective, we do not want to have separate Dockerfiles, or we're not validating the production pipe.

https://github.com/openshift/aws-ebs-csi-driver/blob/master/Dockerfile.openshift.rhel7

We can likely do a follow-up pass to find `ci-operator` configurations for repos in `ocp-build-data` that do not have an entry in `images` that references the production Dockerfile and make some noise to get them to fix it.